### PR TITLE
Wdi14

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

**Comment by nayana487** [comment]
looking!

File: readme.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Create an app that demonstrates your subject. It should...
 
 ### Document It
```
Comment (nayana487) [inline readme.md]: should we put the `Grouping` section higher up? that way, here, you can specify that they only need to submit one repo.

File: readme.md [hunk 2]
```diff
@@ -68,27 +68,15 @@ Additionally, you'll have to do a fraction of the work: one person can work on t
```
Comment (nayana487) [inline readme.md]: ^^ can we kill the line about people completing other work? if they finish early, cool, let them do that. But i don't think we need to be explicit about that in the assignment prompt.  could risk inviting some folks to screw around during the lab and mess up their teammates.

File: readme.md [hunk 3]
```diff
@@ -68,27 +68,15 @@ Additionally, you'll have to do a fraction of the work: one person can work on t
...
-- Any particular sections of your code that you would like the instructors to review, and specific questions to guide our feedback.
+**NO LATER THAN 11:00AM:** Each team must submit a Git issue to the `sweet-libs` repository indicating...
+- What technology you are learning
+- Who is on the team
 
-**3:30pm: ALL STUDENTS** will re-convene in their classroom for presentations.
+**NO LATER THAN 5:00PM:** The issue must be updated to include...
+- A link to your main repository that contains a `readme.md` and code
+- A comfort score (out of 5)
+- A reference to any particular sections of your code that you would like the instructors to review, and specific questions to guide instructor feedback
```
Comment (nayana487) [inline readme.md]: do we want it to be optional to provide questions for instructors to reviews? this reads as if it's mandatory.

File: readme.md (hunk 2 above)
Comment (amaseda) [inline readme.md]: Yup, missed that line. Deleted.

File: readme.md (hunk 3 above)
Comment (amaseda) [inline readme.md]: Sure.

File: readme.md (hunk 1 above)
Comment (amaseda) [inline readme.md]: Moving it up.

**Pull request merged**